Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Powershell_Command_Execution.yml

```diff
@@ -0,0 +1,1481 @@
+id: silent-Suspicious Powershell Command Execution
+version: -1
+issilent: true
+name: silent-Suspicious Powershell Command Execution
+description: |-
+  This playbook handles the following alerts:
+
+  - PowerShell runs suspicious base64-encoded commands
+  - PowerShell suspicious flags
+  - Reverse shell using PowerShell
+  - PowerShell downloads files via BITS
+
+  Playbook Stages
+
+  Analysis:
+
+  - Analyze command-line for suspicious patterns
```

Comment:
- Analyze the command line for suspicious patterns.

Question: is this the order the analysis is performed? If so, it should be a numbered list instead of a bulleted list